doc/specifications/rest/REST-API-V2.rst

```diff
@@ -1454,6 +1455,24 @@ Delete Content Version
     :401: If the user is not authorized to delete this version
     :403: If the version is in state published
 
+Delete Published Content Version Translation
+``````````````````````
+:Resource: /content/objects/<ID>/currentversion/translations/<languageCode>
+:Method: DELETE
+:Description: The translation is removed from a published version (creates and publishes new version)
```

Comment:
Nitpick: I'd rephrase that to "Removes a translation from a published version, and publishes a new version".